### PR TITLE
styleguides and issue

### PR DESCRIPTION
**Pull request by justnpT**
I removed the closed related-issue and updated the class according to Java Styleguide and QA tech meeting talks

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java [hunk 1]
```diff
@@ -21,14 +21,14 @@
...
 
   Credentials credentials = Configuration.getCredentials();
 
 
-  @RelatedIssue(issueID = "SERVICES-439", comment = "Set new password dialog is missing. Please test manually.")
-  @Test(groups = {"ForgottenPassword_001", "ForgottenPassword"})
-  public void ForgottenPassword_001_dropdown() {
+  @Test
+  public void forgottenPassword_001_dropdown() {
     String userName = credentials.userNameForgottenPassword;
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java]: how about
```
@Test
public void remindPasswordFromLoginDrobdown() {
...
@Test
public void remindPasswordOnSpecialPage() {
}
...
@Test
public void remindPasswordWhileCreatingWIki() {
}
...
```
???

Comment (justnpT) [inline src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java]: yes it's better, introduced

**Comment by ludwikkazmierczak** [comment]
LGTM

**Comment by tomasznapieralski** [comment]
LGTM

File: src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java [hunk 2]
```diff
@@ -21,14 +21,14 @@
  * @author Bogna 'bognix' Knychala
  * @author Karol 'kkarolk' Kujawiak
  */
+@Test(groups = {"ForgottenPassword"})
```
Comment (drets) [inline src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java]: `{}` - redundant here

**Pull request merged**